docs/api/rasa-sdk.rst

```diff
@@ -0,0 +1,90 @@
+:desc: Information about `rasa-sdk` objects and functions.
```

Comment:
Can you break out your inner marketer to try to fancy this up? It's for SEO optimization so something like "Customize your Rasa assistant and connect to external APIs using the Rasa-SDK package to write custom actions in Python" or something like that 😄

Reply:
Didn't know this was for SEO
😂 I'll try to make it flashy, add all the buzzwords I can think of